src/UI/Component/Chart/Speedo/Factory.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/* Copyright (c) 2017 Ralph Dittrich <dittrich@qualitus.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Chart\Speedo;
+
+/**
+ * Interface Factory
+ * @package ILIAS\UI\Component\Chart\Speedo
+ */
+interface Factory {
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *     The standard speedo is used if the speedo is to be rendered in it's original size, even on mobile devices.
+     *   composition: >
+     *     The standard speedo is composed of one bar representing a score achieved in relation to a maximum

```

Comment:
This whole paragraph is unintelligible to me. Where does the minimum value come from. It was not introduced before. What is a diagnostic value, where is this introduced? 
There was no mention of a "pointer" so far.

Reply:
It's clear to me. How else would you describe the composition? Minimum is introduced here. Where else would you introduce it? Note that is is reused in the composition section.